keylime/utils/algorithms.py

```diff
@@ -0,0 +1,60 @@
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright 2017 Massachusetts Institute of Technology.
```

Comment:
@kaifeng you can put you're own name there is you like.  May as well make it 2020 as well (as its a new file).

Reply:
okay, just haven't seen many personal copyright in opensource projects.